app/main/forms.py

```diff
@@ -853,6 +853,11 @@ def validate_address(self, field):
                 f'Last line of the address must be a real UK postcode'
             )
 
+        if address.has_invalid_characters:
+            raise ValidationError(
+                'Address lines must not start with any of the following characters: @ ( ) = [ ] ” \\ / ,'
```

Comment:
It would be nice to copy ‘the too many lines’ error and not hard code the characters:
```suggestion
                f'Address lines must not start with any of the following characters: {" ".join(PostalAddress.INVALID_CHARACTERS)}'
```

…but maybe not worth it since:
- we don’t anticipate these characters changing often
- that would mean releasing utils again because the order of the characters in `PostalAddress.INVALID_CHARACTERS` is different 
- doing `join` inside an f-string is arguably a bit gross